### PR TITLE
Whitelist all domains for image

### PR DESCRIPTION
**Pull request by LukasReschke**
Fixes https://github.com/owncloud/files_pdfviewer/issues/63 and https://github.com/owncloud/files_pdfviewer/issues/60 and https://github.com/owncloud/files_pdfviewer/issues/66

### CODE REVIEW

**Comment by LukasReschke** [comment]
```
PHP Fatal error:  Uncaught exception 'OCP\AutoloadNotAllowedException' with message 'Autoload path not allowed: /home/travis/build/owncloud/core/tests/lib/testcase.php' in /home/travis/build/owncloud/core/lib/autoloader.php:134
Stack trace:
#0 /home/travis/build/owncloud/core/lib/autoloader.php(154): OC\Autoloader->isValidPath('/home/travis/bu...')
#1 [internal function]: OC\Autoloader->load('Test\\TestCase')
#2 /home/travis/build/owncloud/core/apps/files_pdfviewer/tests/unit/controller/displaycontrollertest.php(19): spl_autoload_call('Test\\TestCase')
#3 phar:///home/travis/.phpenv/versions/5.5.21/bin/phpunit/phpunit/Util/Fileloader.php(58): include_once('/home/travis/bu...')
#4 phar:///home/travis/.phpenv/versions/5.5.21/bin/phpunit/phpunit/Util/Fileloader.php(42): PHPUnit_Util_Fileloader::load('/home/travis/bu...')
#5 phar:///home/travis/.phpenv/versions/5.5.21/bin/phpunit/phpunit/Framework/TestSuite.php(328): PHPUnit_Util_Fileloader::checkAndLoad('/home/travis/bu...')
#6 phar:///home/travis/.phpenv/versions/5.5.21/bin/phpu in /home/travis/build/owncloud/core/lib/autoloader.php on line 134
```

@Xenopathic :speak_no_evil:

**Comment by Xenopathic** [comment]
@LukasReschke Please read the mailing list and/or the documentation: https://doc.owncloud.org/server/8.2/developer_manual/app/testing.html#php :smile:

TL;DR: You need `\OC::$loader->addValidRoot(\OC::$SERVERROOT . '/tests');` in your test bootstrap.

**Comment by LukasReschke** [comment]
*/me is too stupid* :see_no_evil:

**Comment by volkswagner** [comment]
Thank you for this fix. Now all my pdf files are viewable in owncloud server. No more blank pdf files.
You guys rock!
